kraken-build/tests/kraken_std/integration/cargo/test_cargo_private_registry.py

```diff
@@ -181,9 +181,7 @@ def private_registry(docker_service_manager: DockerServiceManager) -> str:
         detach=True,
     )
 
-    host = "localhost"
-    port = container.ports["35504/tcp"][0]["HostPort"]
-    index_url = f"http://{host}:{port}/git"
+    index_url = f"http://0.0.0.0:35504/git"
```

Comment:
We hardcode the port (see line 180) and the host (line 178), there is no point to do introspection